www/src/pages/components/list-group.mdx

```diff
@@ -78,6 +80,22 @@ edge-to-edge in a parent container [such as a `Card`](/components/cards/#list-gr
   exampleClassName={styles.listGroup}
 />
 
+### Horizontal
+
+Use the `horizontal` prop to make the ListGroup render horizontally. Currently **horizontal list groups cannot be combined with flush list groups.**
```

Comment:
Are we actually enforcing this in the implementation? We should at least log a warning when the end user tries to do this.

Reply:
No, I haven't actually implemented it - do you want me to throw an error, or just log the warning? The "unsupportedness" mostly comes from clashing CSS styling.

Reply:
Also, is there a specific logger I should use? Or just `console.log()`?

Reply:
> does it make more sense to throw an error, or just log the warning? The "unsupportedness" mostly comes from clashing CSS styling.

IMO a warning should be good enough, I'm not sure if a hard fail would provide any more benefit.

> Also, is there a specific logger I should use? Or just console.log()?

We log warnings via the `warning` package. There's an example of its usage in the [FormLabel](https://github.com/react-bootstrap/react-bootstrap/blob/d6026eb0f4c7db98cc676dab119f8c4715775f6a/src/FormLabel.js#L62-L65) implementation.

Reply:
Gotcha, thanks for the pointer! I added a warning on both being used, and added a test in `ListGroupSpec.js` - let me know if it makes sense!